opimodel/fonts.py

```diff
@@ -12,11 +12,15 @@
 
 class Font(object):
 
-    def __init__(self, fontface, size, style=REGULAR, pixels=True):
+    def __init__(self, name=None, fontface='Liberation Sans',
```

Comment:
The advantage of having a default of `name="Default"` is that will give the standard font in whatever environment it is used. 

If the fontface etc are ignored when a font 'name' is specified do you need default values for these too? Can you put placeholder or empty text in these properties? If you put specify `fontface='Arial'` and `name='Default'` your OPI file becomes confusing to anyone reading it as we'd see Liberation Sans.